package.json

```diff
@@ -74,7 +75,9 @@
     ],
     "examples/**/*.html": [
       "npm run reference-tables",
-      "git add examples/index.html"
+      "git add examples/index.html",
+      "npm run coverage-report",
+      "git add coverage/"
```

Comment:
@howard-e are these explicit `git add` still ok with the new lint-staged?

Reply:
Those should be fine